packages/sdks/snippets/remix/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@snippet/remix",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "build": "react-router build",
+    "dev": "react-router dev",
+    "start": "react-router-serve ./build/server/index.js",
```

we need a `serve` command for tests to run
```suggestion
    "start": "react-router-serve ./build/server/index.js",
    "serve": "yarn start",
```